app/components/Home.tsx

```diff
@@ -25,7 +25,7 @@ const MarkdownParser = MarkdownIt(markdownitOptions)
 type AppProps = {
   text?: string,
   mode: any;
-  platform?: any,
+  platform?: 'desktop' | 'mobile',
```

Comment:
Me saying "desktop | mobile" was just an example. I don't believe that is the proper range of value.

https://github.com/standardnotes/web/blob/develop/app/assets/javascripts/utils.js#L23

I believe they are:
[web/desktop]-mac
[web/desktop]-linux
[web/desktop]-windows
android
ios

Reply:
Should we use `platform?: string` since we don't need to test for specific values?